core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -696,8 +696,11 @@ private[spark] class DAGScheduler(
     val jobId = nextJobId.getAndIncrement()
     if (partitions.isEmpty) {
       val time = clock.getTimeMillis()
+      val dummyStageInfo = new StageInfo(
+        -1, -1, callSite.shortForm, 0, Seq.empty[RDDInfo], Seq.empty[Int], "")
```

Comment:
Can we have a CONSTANT for this `-1`?

Reply:
How about defining `INVALID_STAGE_ID` and `INVALID_ATTEMPT_ID` in `StageInfo`?

Reply:
+1. Thanks~